spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/resolver/AetherModuleResolver.java

```diff
@@ -88,10 +89,10 @@ public AetherModuleResolver(File localRepository, Map<String, String> remoteRepo
 		}
 		this.localRepository = localRepository;
 		this.remoteRepositories = new LinkedList<>();
-		if (!CollectionUtils.isEmpty(remoteRepositories)) {
-			for (Map.Entry<String, String> remoteRepo : remoteRepositories.entrySet()) {
-				RemoteRepository remoteRepository = new RemoteRepository.Builder(remoteRepo.getKey(),
-						DEFAULT_CONTENT_TYPE, remoteRepo.getValue()).build();
+		if (!CollectionUtils.isEmpty(remoteRepositoriesSet)) {
+			for (String remoteRepo : remoteRepositoriesSet) {
+				RemoteRepository remoteRepository = new RemoteRepository.Builder("",
```

Comment:
I think that we should generate a unique id per repo here. I don't think it will have a huge functional impact right now, but it may affect things like adding credentials.